From 502ebf6397854eaf14bd9bd00093dc3df1cdacfb Mon Sep 17 00:00:00 2001
From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
Date: Tue, 29 Aug 2023 12:12:15 +0530
Subject: [PATCH] VTS fix added additional nodes for usb-passthrough

Added genfs rules for ADL-PS, ADL-P and ADL-S devices
With USB-PASSTRHOUGH showing ACPI LNXSYSTM/LNXSYBUS and USB wake nodes

fix VTS test SuspendSepolicyTests

Tracked-On: OAM-111651
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>

diff --git a/vendor/genfs_contexts b/vendor/genfs_contexts
index 8a498ec..b2a6c05 100644
--- a/vendor/genfs_contexts
+++ b/vendor/genfs_contexts
@@ -56,5 +56,14 @@ genfscon sysfs /devices/platform/rtc_cmos/rtc/rtc0/alarmtimer.1.auto/wakeup u:ob
 genfscon sysfs /devices/platform/rtc_cmos/rtc/rtc0/alarmtimer.2.auto/wakeup u:object_r:sysfs_wakeup:s0
 genfscon sysfs /devices/platform/rtc_cmos/rtc/rtc0/alarmtimer.3.auto/wakeup u:object_r:sysfs_wakeup:s0
 
+genfscon sysfs /devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:10/wakeup u:object_r:sysfs_wakeup:s0
+genfscon sysfs /devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:0d/wakeup u:object_r:sysfs_wakeup:s0
+genfscon sysfs /devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:0e/wakeup u:object_r:sysfs_wakeup:s0
+genfscon sysfs /devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:0f/wakeup u:object_r:sysfs_wakeup:s0
+
+genfscon sysfs /devices/pci0000:00/0000:00:07.0/usb1/1-2/wakeup u:object_r:sysfs_wakeup:s0
+genfscon sysfs /devices/pci0000:00/0000:00:0d.0/usb5/5-2/wakeup u:object_r:sysfs_wakeup:s0
+genfscon sysfs /devices/pci0000:00/0000:00:08.0/wakeup u:object_r:sysfs_wakeup:s0
+
 genfscon proc  /sys/vm/swappiness        u:object_r:proc_swappiness:s0
 genfscon proc  /sys/vm/disk_based_swap   u:object_r:proc_disk_based_swap:s0
-- 
2.34.1

